lib/gui/components/drive-selector/controllers/drive-selector.js

```diff
@@ -28,6 +34,13 @@ module.exports = function($q, $uibModalInstance, DrivesModel, SelectionStateMode
   this.state = SelectionStateModel;
 
   /**
+   * @summary Static methods to check a drive's properties
+   * @property
+   * @type Object
+   */
+  this.constraints = DriveConstraintsModel;
```

Comment:
I wonder if we can require this particular module in the places that use this logic instead of completely exposing it here. As a bonus, the Redux store has had TODO comments to re-use these static functions for a while: https://github.com/resin-io/etcher/blob/master/lib/gui/models/store.js#L99